### DISCUSSION TITLE
Is it possible to configure gaps per workspace?

### DISCUSSION THREAD

**Discussion by Hubro** [hyprwm/Hyprland#2076] [Q&A] [answered]
I'm coming from Sway, and there I usually disabled gaps for a couple of my workspaces because I only ever have 1 window running on them. For example, on workspace 6 I only run games. For the games I run in windowed mode, I want the game window to cover the entire screen except for Waybar, i.e. no gaps.

After skimming through the wiki, I can't find a way to do this. Is it possible in Hyprland?

**Comment by vaxerski** [comment, marked as answer]
go to your layout's page and look for no gaps when only

**Reply by Hubro** [reply]
Not quite the same, since I generally still want the gaps (it looks nice!) even when I only have a single window on the screen, there are just some exceptions like games where I want the window to cover the entire screen.

This is much better than nothing though, so I have enabled it :slightly_smiling_face: Thanks

As you are the author, I take this to mean that it's not possible to configure gaps per workspace.

**Reply by vaxerski** [reply]
it's not. There is an MR in the works to allow that though

**Reply by Hubro** [reply]
> it's not. There is an MR in the works to allow that though

Very cool!

Is it already possible to set different layouts for each workspace, or is that something that could possibly be added to the PR?

**Reply by vaxerski** [reply]
layouts per WS are not coming any time soon

**Reply by JakeTompkins** [reply]
Is that a matter of technical limitation of just priorities? Would you accept a PR?

**Reply by vaxerski** [reply]
the current system is simply completely not made for that

**Reply by stefanmrvic** [reply]
Hey, @vaxerski will this change down the line? (if it already hasn't)

**Reply by vaxerski** [reply]
no. Not for the forseeable future.

**Reply by stefanmrvic** [reply]
Another thing that I'm trying to achieve is when I have floated window covering tiling window(s), to be able to left click tiling window and floating window to go behind the tiling window that I've clicked. Is this possible to achieve? If so how? I have tried looking at Wiki and Reddit, but was unable to find the solution. @vaxerski
